docs/user_guide/event_processors/event_rate_limit.md

```diff
@@ -0,0 +1,28 @@
+The `event-rate-limit` processor rate-limits each event with matching tags to the configured amount per-seconds.
+
+All the tags for each event is hashed, and if the hash matches a previously seen event, then the timestamp 
+of the event itself is compared to assess if the configured limit has be exceeded.
```

Comment:
Minor typo: `limit has be exceeded.` --> `limit has been exceeded`

Reply:
done